app/test_util.py

```diff
@@ -50,13 +50,13 @@ def wait_for_css(self, css, timeout=5):
         )
 
     def wait_for_xpath_to_disappear(self, xpath, timeout=5):
-        return WebDriverWait(self, timeout).until_not(
-            expected_conditions.presence_of_element_located((By.XPATH, xpath))
+        return WebDriverWait(self, timeout).until(
+            expected_conditions.invisibility_of_element((By.XPATH, xpath))
         )
 
     def wait_for_css_to_disappear(self, css, timeout=5):
         return WebDriverWait(self, timeout).until_not(
-            expected_conditions.presence_of_element_located((By.CSS, css))
+            expected_conditions.invisibility_of_element((By.CSS, css))
```

Comment:
This is `until_not` and `invisibility` -- should it be `until` above?

Reply:
Correct!